fern/docs.yml

```diff
@@ -19,85 +19,71 @@ tabs:
   home:
     display-name: Docs
     icon: home
-  guides:
-    display-name: Guides
-    icon: book
   API Reference:
     display-name: API Reference
     icon: puzzle
 
 navigation:
   - tab: home
     layout:
-      - section: Get Started
+      - section: Get started
         contents:
           - page: Welcome
             path: docs/pages/welcome.mdx
-  - tab: guides
-    layout:
-      - section: Get started
-        contents:
-          - page: Overview
-            path: docs/pages/get-started/overview.mdx
-            icon: fa-duotone fa-book-open
-          - page: Quickstart
-            path: docs/pages/get-started/quickstart.mdx
-            icon: fa-duotone fa-rocket
-      - section: Capabilities
-        contents:
-          - page: Plant management
-            path: docs/pages/capabilities/plant-management.mdx
-            icon: fa-duotone fa-seedling
-          - page: Order processing
-            path: docs/pages/capabilities/order-processing.mdx
-            icon: fa-duotone fa-cart-shopping
-          - page: Customer management
-            path: docs/pages/capabilities/customer-management.mdx
-            icon: fa-duotone fa-users
-          - page: Inventory tracking
-            path: docs/pages/capabilities/inventory-tracking.mdx
-            icon: fa-duotone fa-boxes-stacked
-      - section: Tutorials
-        contents:
-          - page: Searching for plants
-            path: docs/pages/tutorials/searching-plants.mdx
-            icon: fa-duotone fa-magnifying-glass
-          - page: Creating your first order
-            path: docs/pages/tutorials/first-order-tutorial.mdx
-            icon: fa-duotone fa-bag-shopping
-          - page: Managing plant inventory
-            path: docs/pages/tutorials/inventory-tutorial.mdx
-            icon: fa-duotone fa-clipboard-list
+            icon: fa-duotone fa-house
+          - page: Edit your docs
+            path: docs/pages/editing-your-docs.mdx
+            icon: fa-duotone fa-pen-to-square
+          - page: Write content
+            path: docs/pages/writing-content.mdx
+            icon: fa-duotone fa-file-lines
+          - page: Set up navigation
+            path: docs/pages/navigation.mdx
+            icon: fa-duotone fa-sitemap
+          - page: Customize your docs
+            path: docs/pages/customization.mdx
+            icon: fa-duotone fa-palette
+          - page: Support
+            path: docs/pages/support.mdx
+            icon: fa-duotone fa-headset
       - section: Changelog
         contents:
           - changelog: docs/changelog
   - tab: API Reference
```

Comment:
@sbawabe want to add:
- OpenAPI overrides (estimate of 30% of users use it)
- an AsyncAPI spec (estimate of 10% of users use it)